### PR TITLE
Feature 211 refine sosmeta translation

### PR DESCRIPTION
**Pull request by Jdallos**
-Empty sections removed and missing henkilötunnus fields now translated

### CODE REVIEW

File: src/utils/sosmeta-utils.ts [hunk 1]
```diff
@@ -72,11 +76,19 @@ namespace SosmetaUtils {
...
-              field.properties[x]
-            ));
+          // eslint-disable-next-line no-case-declarations
+          const firstKey = Object.keys(field.properties)[0];
+          if (field.properties[firstKey].sosmeta) {
+            return Object.keys(field.properties).map(x =>
+              handleSosmetaStringField(
+                sectionName,
+                field.properties[x]
+              ));
```
Comment (EVP1989) [inline src/utils/sosmeta-utils.ts:86]:
```suggestion
            return Object.keys(field.properties).map(property =>
              handleSosmetaStringField(
                sectionName,
                field.properties[property]
              ));
```

File: src/utils/sosmeta-utils.ts [hunk 2]
```diff
@@ -160,9 +172,13 @@ namespace SosmetaUtils {
...
    */
   const handleSosmetaBooleanSection = (sosmetaSection: any) => {
-    const { required } = sosmetaSection;
+    try {
+      const { required } = sosmetaSection;
 
-    return [ handleSosmetaBooleanField(sosmetaSection.sosmeta.name[0].value, sosmetaSection, required) ];
+      return [ handleSosmetaBooleanField(sosmetaSection.sosmeta.name[0].value, sosmetaSection, required) ];
+    } catch (e) {
+      // TODO: Error handling/messagess
```
Comment (EVP1989) [inline src/utils/sosmeta-utils.ts:180]: Maybe error handling could be just console.error(e) in these cases.

**Review by EVP1989 — APPROVED** [review]
Approved with comments

File: src/utils/sosmeta-utils.ts (hunk 2 above)
Comment (Jdallos) [reply src/utils/sosmeta-utils.ts]: Done, for all sosmeta utils. Now its clearer that there are some errors when we translate certain sosmeta forms, but this is working as before.

**Review by EVP1989 — APPROVED** [review]
Approved

**Pull request merged**